### PR TITLE
Header and first section

### PR DESCRIPTION
**Pull request by PedroH-Neves**
- Imported google fonts and assets
- Created mobile version of header and introduction section
- Add CSS styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  <title>Portfolio</title>
+</head>
+<body id="body">
+    <header id="header">
+      <p class="logo primary-color">
+        P
+      </p>
+      <div class="menu-icon">
+        <img src="./assets/Icons/Burger.svg" alt="menu icon">
+      </div>
```
Comment (ghost) [inline index.html]: - [ ] Would you please wrap your hamburger button with a `<nav>` tag. This is for semantic and accessibility reasons

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Rubik+Maze&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+<body id="body">
+    <header id="header">
+      <p class="logo primary-color">
```
Comment (ghost) [inline index.html]: - [ ] Would you please wrap your logo in an `<a>`  tag.  This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        P
+      </p>
+      <div class="menu-icon">
+        <img src="./assets/Icons/Burger.svg" alt="menu icon">
+      </div>
+    </header>
+    <div class="section-one background">
```
Comment (ghost) [inline index.html]: - [ ] Would you please wrap your headline section in a `<section>` tag. This is important for semantic and accessibility reasons

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

- Have used the right GitHub flow ✅
- Meaningful  summary for PR ✅
- Linter checks passed ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi, @PedroH-Neves  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**